docs/UsingNavigators.md

```diff
@@ -103,7 +103,7 @@ A more complete example that demonstrates the pushing and popping of routes coul
 
 ```javascript
 import React, { Component, PropTypes } from 'react';
-import { Navigator, Text, TouchableHighlight, View } from 'react-native';
+import {AppRegistry, Navigator, Text, TouchableHighlight, View } from 'react-native';
```

Comment:
nit: space between `{` and `AppRegistry`.

I'd rather add `AppRegistry` in the places where it's used (it's not used in the code snippet where you added it)